contracts/scripts/update-round.ts

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs';
+import hre from 'hardhat';
+import { create } from 'ipfs-http-client';
+import { ScriptLogger } from './ScriptLogger';
+import params from './update-round.config';
+
+const { ethers } = hre;
+
+const ipfs = create({
+  url: 'https://ipfs-api.dev.fleek.cool',
+  headers: {
+    Authorization: `v2 ${process.env.FLEEK_STORAGE_API_KEY}`,
+  },
+});
```

Comment:
`ipfs` instances are created like this in 3 separate places. Feels like a good candidate to pull out into a utility method in `@dgrants/utils`, with the method's input being the Fleek API key

Reply:
Great call. Done in a60210972147240914729f51cff673a853c043f7